pkgs/os-specific/bsd/openbsd/pkgs/libc/libc-no-tags.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/lib/libc/Makefile b/lib/libc/Makefile
+index 4bb4b67fcbb..daa9275d318 100644
+--- a/lib/libc/Makefile
++++ b/lib/libc/Makefile
+@@ -26,6 +26,10 @@ SYMBOL_LISTS=	${LIBCSRCDIR}/Symbols.list \
+ LIBCSRCDIR=${.CURDIR}
+ .include "${LIBCSRCDIR}/Makefile.inc"
+ 
++all: ${SRCS}
++
++LIBC_TAGS ?= yes
```

Comment:
Why `?=`?

Reply:
To hint that it's just a default. I vaguely recall `?=` isn't as useful for BSD Make as it is for GNU Make, but I forget the details.

Reply:
Every make variable that doesn't use := is just a default.  The only different the ? makes is that it will prefer a value set in the environment (rather than as a make flag) to the default, which IMO is quite fragile unless it's a conventional environment variable like CC.

Reply:
Ah right, thanks!